### ISSUE TITLE
TextureAtlas keep re-importing on every startup

### ISSUE THREAD

**Issue by boruok** [godotengine/godot#66529]
### Godot version

Godot_v4.0-beta1_win64

### System information

Windows 11, GTX1060, Vulkan

### Issue description

Tried to generate TextureAtlas from separate textures, on 3.x branch atlas generation happens only once, 
 in 4.x atlas auto generated each time when opening project.

### Steps to reproduce

1. open attachment, notice that textures are reimporting
2. reload current project
3. textures are reimporting...

### Minimal reproduction project

[New Game Project.zip](https://github.com/godotengine/godot/files/9662648/New.Game.Project.zip)

**Comment by Daylily-Zeleen** [comment]
I meet this bug, too.
It's relate about multi threads importing.
In 3.x, default is single thread import. But in 4.0, default is multi threads import.

**Comment by felbecker** [comment]
To keep things together: This is a duplicate of #54817 and apparently already fixed by #60888.
